### ISSUE TITLE
Suricata Script does not work

### ISSUE THREAD

**Issue by himuura** [pwnlandia/mhn#836]
Hey!
Been testing some probes and suricata script doesn't seem to work due to the fact that the suricata rules do not exist anymore on that link, it needs to be update. Im testing with another suricata version but i'm not confident...
Anyone to help a brother out?
Thanks in advance!
(using ubuntu 16.04)

**Comment by wrharding** [comment]
Can you provide an output of the failure?

**Comment by wrharding** [comment]
Related: #798

**Comment by himuura** [comment]
Exactly! It was this exact error mate, sorry for the delay but i left the lab some hours ago. Gonna try the sed to replace the version of suricata.
One other thing, setuptools fails to install on some scripts. I managed to install cowrie for instance without setuptools (supervisor) and it seems to be working...The same happened with suricata. Had to comment out pyyaml and i think distribute as well so i could bypass that part of the script. Any suggestions on how to fix it?
